docs: add note about windows+powershell (#323)
clinicjs/node-clinic: README.md

utilize more than 100%. The visible percentage is always the combination of all
these factors together.

### Windows + PowerShell

> In order to diagnostic your application with node clinic, is mandatory execute your application after double hyphen(`--`),

Reviewer: ```suggestion
In order to diagnose your application with node clinic, you should execute your application after double hyphens (`--`),
```

e.g: `clinic doctor -- node myapplication.js`.

However, if you are using Windows and PowerShell need to know that a double hyphen instructs PowerShell to

> treat everything coming after as literal arguments rather than options.

Reviewer: the english is not great here, rephrase

Author: I have slightly changed, could you check?

> To avoid that behavior you can either quote ("--", '--') or escape (`--) the double hyphen

Reviewer: we're missing a backtick here
